framework/src/main/kotlin/dev/alpas/view/ViewRenderer.kt

```diff
@@ -6,15 +6,15 @@ import dev.alpas.validation.SharedDataBag
 
 interface ViewRenderer {
     fun extend(extension: AbstractExtension, vararg extensions: AbstractExtension)
-    fun render(viewName: String?, sharedDataBag: SharedDataBag, viewArgs: Map<String, Any?>? = null): String {
+    fun renderView(viewName: String?, sharedDataBag: SharedDataBag, viewArgs: Map<String, Any?>? = null): String {
         TODO()
     }
 
-    fun render(template: String, args: Map<String, Any?>? = null): String {
+    fun renderTemplate(template: String, sharedDataBag: SharedDataBag, args: Map<String, Any?>? = null): String {
```

Comment:
Are we okay with adding SharedDataBag here? There were a few existing calls that needed a SharedDataBag but the first argument was a template and not a view name

Reply:
Yes. I think it is more consistent.